doc/pages/FAQ.md

```diff
@@ -62,6 +62,35 @@ remove) are also run in a sandbox and guaranteed not to affect your system.
 
 ---
 
+#### 🐫  Why does opam require ``bwrap`` ?
+
+Since opam 2.0.0~rc2, opam uses `bwrap` on Linux to run package instructions in
+a sandbox. See the [bubblewrap page](https://github.com/projectatomic/bubblewrap)
```

Comment:
Excellent FAQ! Just one thing that might be worth explaining a bit more: what a sandbox is, and how opam uses it.  We use bwrap to prevent packages from writing outside their allotted filesystem space within the `OPAMROOT`, but do not prevent network access.